docs/Dev_eXo_Addons.rst

```diff
@@ -1139,11 +1150,188 @@ following:
+        <component-plugin>
+          <!-- One of the listed event names -->
+          <name>EVENT_NAME</name>
+          <set-method>addListener</set-method>
+          <!-- FQN of the event listener -->
+          <type>org.example.CustomEventListener</type>
+        </component-plugin>
+      </external-component-plugins>
+
+.. _PLFDevGuide.eXoAdd-ons.KudosListeners:
+
+==============================
+eXo Kudos listeners
+==============================
+
+eXo Kudos uses :ref:`ListenerService <Kernel.UnderstandingtheListenerService>` to broadcast events about kudos lifecycle.
+Developers can leverage these events to build custom features or alter external systems.
+Internally, these events are used for a variety of uses such as creating an activity or comment or attribute gamification points for kudos receiver and sender.
+
+Broadcasted events are:
+
+- ``exo.addons.kudos.sent`` : a new Kudos is sent. (Example: `NewKudosSentActivityGeneratorListener <https://github.com/exoplatform/kudos/blob/develop/kudos-services/src/main/java/org/exoplatform/addon/kudos/listener/NewKudosSentActivityGeneratorListener.java>`__)
+- ``exo.addons.kudos.activity`` : a Kudos activity or activity comment is created. (Example: `GamificationIntegrationListener <https://github.com/exoplatform/kudos/blob/develop/kudos-services/src/main/java/org/exoplatform/addon/kudos/listener/GamificationIntegrationListener.java>`__)
+
+To add an event listener using one listed events above, you can add the following configuration inside a  :ref:`Portal extension <PLFDevGuide.eXoAdd-ons.PortalExtension>` configuration file:
+
+   .. code:: xml
+
+      <external-component-plugins>
+        <target-component>org.exoplatform.services.listener.ListenerService</target-component>
+        <component-plugin>
+          <!-- One of the listed event names -->
+          <name>EVENT_NAME</name>
+          <set-method>addListener</set-method>
+          <!-- FQN of the event listener -->
+          <type>org.example.CustomEventListener</type>
+        </component-plugin>
+      </external-component-plugins>
+
+.. _PLFDevGuide.eXoAdd-ons.PerkStoreListeners:
+
+==============================
+eXo Perk Store listeners
+==============================
+
+eXo Perk Store uses :ref:`ListenerService <Kernel.UnderstandingtheListenerService>` to broadcast events about products and orders lifecycle.
+Developers can leverage these events to build custom features or alter external systems.
+Internally, these events are used for a variety of uses such as notifications or updating UI in realtime using Websocket.
+
+Broadcasted events are:
+
+- ``exo.perkstore.settings.modified`` : Perk Store settings gets modified by an administrator. (Example: `WebSocketSettingsListener <https://github.com/exoplatform/perk-store/blob/develop/perk-store-services/src/main/java/org/exoplatform/addon/perkstore/listener/WebSocketSettingsListener.java>`__)
```

Comment:
gets=>get

Reply:
http://conjugator.reverso.net/conjugation-english-verb-get.html (It gets)
I will use the same conjugation as the two events listed below. (is modified)

Reply:
done

Reply:
settings are plural, that's why theu is used instead of it :)